config/locales/de.yml

```diff
@@ -47,56 +47,56 @@ de:
         precision: 0
         format: '%u%n'
   homepage:
-    intro: "<strong>codebar</strong> is a charity that facilitates the growth of a diverse tech community by running free regular programming workshops for minority groups in tech."
+    intro: "<strong>codebar</strong> ist eine Wohltätigkeitsorganisation, die das Wachstum einer vielfältigen Tech-Community fördert, indem sie regelmäßig kostenlose Programmier-Workshops für Minderheiten in der Tech-Branche anbietet."
     explanation: "Unser Ziel ist unterrepräsentierten Menschen zu programmieren in einer sicheren und kollaborative Umgebung lernen und erweitern ihre Karrierechancen zu ermöglichen. Um dies zu erreichen betreiben wir kostenlose regelmäßige Workshops, regelmäßige Einzelveranstaltungen und versuchen, die Möglichkeiten für unsere Schüler die Technologie und Programmierung zugänglicher zu schaffen."
     participate:
-      students_title: "Students"
-      students_explanation_p1: "Our students come from a variety of backgrounds. Some want to become full-time developers, whereas some would like to learn the basics of coding in a supportive environment."
-      students_explanation_p2: "Sign up to join our vibrant community and attend our workshops."
-      students_button: "Sign up as a student"
+      students_title: "Studenten"
+      students_explanation_p1: "Unsere Studenten kommen aus den unterschiedlichsten Bereichen. Einige wollen Vollzeitentwickler werden, während andere die Grundlagen des Programmierens in einem unterstützenden Umfeld erlernen möchten."
```

Comment:
`Vollzeitentwickler` -> `Entwickler in Vollzeit`, but that's very subjective. Nothing objectively wrong here.

Reply:
I agree with @m90, but would suggest an inclusive term ,e.g. `Entwickler:innen in Vollzeit`. I don't know how others feel about it, but the `:` is my current go to to make gendered nouns inclusive.